components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/user/OpenIDConnectUserEndpointTest.java

```diff
@@ -82,10 +84,14 @@
 
     private OpenIDConnectUserEndpoint openIDConnectUserEndpoint;
 
+    private MultivaluedMap<String, String> paramMap;
+
     @BeforeTest
     public void setUp() throws Exception {
 
         openIDConnectUserEndpoint = new OpenIDConnectUserEndpoint();
+        paramMap = new MultivaluedHashMap<>();
```

Comment:
Can we have a unit test covering the scenario where the access token comes in the body as a form param